sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/Models/BatchAccountIdentity.cs

```diff
@@ -0,0 +1,86 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Batch.Models
+{
+    using Newtonsoft.Json;
+    using System.Linq;
+
+    /// <summary>
+    /// The identity of the Batch account, if configured. This is only used
+    /// when the user specifies 'Azure.KeyVault' as their Batch account
```

Comment:
This is a typo in the Swagger specification. This should read `Microsoft.KeyVault`. The `EncryptionProperties` object does not have an enum value of `Azure.KeyVault`.